packages/salesforcedx-vscode-core/src/context/workspaceContext.ts

```diff
@@ -43,7 +43,7 @@ export class WorkspaceContext {
   }
 
   protected async handleCliConfigChange(orgInfo: OrgUserInfo) {
-    workspaceContextUtils.setupWorkspaceOrgType(orgInfo.username).catch(e =>
+    setupWorkspaceOrgType(orgInfo.username).catch(e =>
```

Comment:
why'd you change this?

Reply:
Was probably a misguided dev attempt while trying to get the unit tests running.  I conflate this one with the workspaceContext instance issue sometimes I think.  I set it back.